### PR TITLE
Add a bit of functionality to loadtest

### PR DESCRIPTION
**Pull request by aforge**
Publish to all user topics in the load test config.

### CODE REVIEW

File: loadtest/tsung.xml [hunk 1]
```diff
@@ -43,28 +43,41 @@
...
+      <for from="1" to="100" incr="1" var="ctr">
+        <!-- Iterate over subscriptions -->
+        <foreach name="topicx" in="subs">
+          <thinktime value="3" random="true"></thinktime>
 
-      <request subst="true">
-        <websocket type="message" frame="text">{"sub":{"id":"%%_baseid%%05","topic":"%%_topic%%","get":{"what":"desc sub data del"}}}</websocket>
-      </request>
+          <request subst="true">
+            <websocket type="message" frame="text">{"sub":{"id":"%%_baseid%%%%_topicx%%%%_ctr%%05","topic":"%%_topicx%%","get":{"what":"desc sub data del"}}}</websocket>
```
Comment (or-else) [inline loadtest/tsung.xml:62]: A small nitpick: the `05` in `%%_baseid%%%%_topicx%%%%_ctr%%05` is probably not necessary. The ID is already sufficiently unique.

File: loadtest/tsung.xml [hunk 2]
```diff
@@ -43,28 +43,41 @@
...
-      <for from="1" to="5" incr="1" var="counter">
-        <thinktime min="2" max="10" random="true"></thinktime>
+          <for from="1" to="50" incr="1" var="counter">
+            <thinktime min="2" max="10" random="true"></thinktime>
 
-        <request subst="true">
-          <websocket type="message" frame="text">{"pub":{"id":"%%_baseid%%06","topic":"%%_topic%%","content":"This is a Tsung test %%_baseid%% %%_counter%%"}}</websocket>
-        </request>
+            <request subst="true">
+              <websocket type="message" frame="text">{"pub":{"id":"%%_baseid%%%%_topicx%%%%_ctr%%06%%_counter%%","topic":"%%_topicx%%","content":"This is a Tsung test %%_baseid%% %%_counter%%"}}</websocket>
```
Comment (or-else) [inline loadtest/tsung.xml:69]: Likewise, `06` is unnecessary unless you like it there for some reason.

File: loadtest/tsung.xml [hunk 3]
```diff
@@ -43,28 +43,41 @@
...
+            </request>
+          </for>
+
+          <request subst="true">
+            <websocket type="message" frame="text">{"leave":{"id":"%%_baseid%%%%_topicx%%%%_ctr%%07","topic":"%%_topicx%%"}}</websocket>
```
Comment (or-else) [inline loadtest/tsung.xml:74]: ditto `07`

File: loadtest/tsung.xml (hunk 1 above)
Comment (aforge) [reply loadtest/tsung.xml]: Done.

File: loadtest/tsung.xml (hunk 2 above)
Comment (aforge) [reply loadtest/tsung.xml]: Done.

File: loadtest/tsung.xml (hunk 3 above)
Comment (aforge) [reply loadtest/tsung.xml]: Done.

**Pull request merged**